tmpfiles.d/provision.conf

```diff
@@ -20,3 +20,4 @@ f^ /etc/hosts - - - - network.hosts
 d- /root :0700 root :root -
 d- /root/.ssh :0700 root :root -
 f^ /root/.ssh/authorized_keys :0600 root :root - ssh.authorized_keys.root
+f^ /run/ssh.authorized_keys.root :0600 root :root - ssh.authorized_keys.root
```

Comment:
I'm not sure why this works then, it worked for me during testing.

I originally had this as part of systemd-ssh-generator but thought that it might be more appropriate as a tmpfiles.d entry.

Reply:
maybe the file gets applied once in the initrd, and then during the main system it's already symlinked. but we shiuldn't rely on that. initrds are optional after all